docs/autoplay.md

```diff
@@ -0,0 +1,82 @@
+## Managing Auto Play
+If you would like to start automatically playing your content, you can easily manipulating the player configuration and set the `playback.autoplay` value accordingly.
+
+>### playback.autoplay
+>##### Type: `boolean`
+>##### Default: `false`
+>##### Description: Start playback automatically.
+>If `true`, playback will start automatically when sources will be given to the player. If `false`, a user gesture will be required to start playback.
+#### Example - Basic usage:
+```js
+var config = {
+  playback: {
+    autoplay: true
+  },
+  sources: {...}
+};
+// No need to add additional code, 
+// playback will start automatically!
+var player = Playkit.loadPlayer(config);
+```
+
+>**_Note_**:
+> Some developers may 
+> simulate the autoplay behavior by calling `play()` method right after the creation of the player without manipulating the player configuration. We strongly recommend **not** doing it this way!
+>```js
+>var config = {
+>  sources: {...}
+>};
+>var player = Playkit.loadPlayer(config);
+>// Bad practice!
+>player.play(); 
+>```
+> #### Why it's bad?
```

Comment:
Why not use this Method?

Reply:
@AnnaAleksandrowicz  see my comment above

Reply:
@dan-ziv - sorry I wasn't commenting - i was suggesting to rename the heading ""Why not use this method?" Sorry if I wasn't clear!